src/components/FormattedAddress.tsx

```diff
@@ -84,14 +84,15 @@ export default function FormattedAddress({
       } catch (e) {
         console.error('Error looking up ENS name for address', address, e)
       }
-
-      window.localStorage?.setItem(
-        getStorageKey(),
-        JSON.stringify({
-          ...getEnsDict(),
-          [_address]: newRecord,
-        }),
-      )
+      if (typeof window !== 'undefined') {
```

Comment:
running `yarn build` would generate null exception errors through out the project. I added window checks (which should get abstracted and cleaned up) so nextjs build doesn't try and access window objects when there's no window to object.